apps/rewards/app/utils/metric-utils.js

```diff
@@ -28,16 +28,22 @@ const calculateMonthlyAvg = (rewards, balances, convertRates) => {
   if(rewards.length === 0) {
     return 0
   }
-  let monthCount = Math.ceil((rewards.reduce((minDate, reward) => {
-    return reward.endDate < minDate.endDate ? reward: minDate
-  }, rewards[0]).endDate - Date.now())/ MILLISECONDS_IN_A_MONTH)
+  const earliestReward = rewards.reduce(
+    (minDate, reward) => {
+      return reward.endDate < minDate.endDate ? reward : minDate
+    },
+    rewards[0]
+  )
```

Comment:
I find this pretty confusing. Maybe we could do something like this instead:
```suggestion
  const earliestEndDate = Math.min(...rewards.map(r => r.endDate))
```